examples/purchaseTesterTypescript/ios/PurchaseTester.xcodeproj/project.pbxproj

```diff
@@ -439,6 +439,7 @@
 			baseConfigurationReference = 5B7EB9410499542E8C5724F5 /* Pods-PurchaseTester-PurchaseTesterTests.debug.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
+				DEVELOPMENT_TEAM = 8SXR2327BM;
```

Comment:
will this be awkward for anyone actually using this to test?

Reply:
It's trivial to change